packages/salesforcedx-vscode-core/src/commands/forceSourceDeployManifest.ts

```diff
@@ -61,22 +68,19 @@ export async function forceSourceDeployManifest(manifestUri: vscode.Uri) {
     }
   }
 
-  const messages: ConflictDetectionMessages = {
-    warningMessageKey: 'conflict_detect_conflicts_during_deploy',
-    commandHint: input => {
-      return new SfdxCommandBuilder()
-        .withArg('force:source:deploy')
-        .withFlag('--manifest', input as string)
-        .build()
-        .toString();
-    }
-  };
-
-  const commandlet = new SfdxCommandlet(
-    new SfdxWorkspaceChecker(),
-    new FilePathGatherer(manifestUri),
-    new LibrarySourceDeployManifestExecutor(),
-    new TimestampConflictChecker(true, messages)
+  const messages:
```

Comment:
The typing here isn't needed, since the construction and assignment are inline and the type inferred by the function called.

```
  const messages = getConflictMessagesFor(
    'force_source_deploy_with_manifest_beta'
  );
```